src/Compilers/CSharp/Portable/Lowering/Instrumentation/Instrumenter.cs

```diff
@@ -228,6 +229,19 @@ public virtual BoundExpression InstrumentCall(BoundCall original, BoundExpressio
             return rewritten;
         }
 
+        /// <summary>
+        /// Similarly to an interceptor, gives the instrumenter an opportunity to adjust call target, receiver and arguments.
+        /// </summary>
+        public virtual void InterceptCallAndAdjustArguments(
+            ref MethodSymbol method,
```

Comment:
>ref MethodSymbol method, [](http://example.com/codeflow?start=12&length=24)

Are we actually planning to change the method?

Reply:
Yes, we may replace it with a different overload.